src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -201,12 +197,12 @@ public Object getTarget() {
 	}
 
 	@Override
-	public AbstractBuild<?,?> getBuild() {
+	public Run<?,?> getBuild() {
```

Comment:
However, it does not seem to be [used outside](https://github.com/search?utf8=%E2%9C%93&q=JacocoBuildAction+getOwner+user%3Ajenkinsci&type=Code&ref=searchresults), but who knows.